chainer/utils/sparse.py

```diff
@@ -0,0 +1,73 @@
+import chainer
+from chainer.backends import cuda
+
+
+class SparseCooMatrix(object):
+
+    def __init__(self, data, row, col, shape, use_variable=False):
```

Comment:
What do you think of always wrapping the data in a `Variable` (instead of `use_variable`) and let the user control the gradient with an additional `require_grad` setter/getter property. That would simplify the interface a bit and make it more similar to the `Variable`. The overhead isn't significant and maybe more future proof?

Reply:
+1 for

> let the user control the gradient with an additional require_grad setter/getter property.

In that case, do we need additional `require_grad` option?